### ISSUE TITLE
Cannot disable for Google Slides

### ISSUE THREAD

**Issue by everyonesdesign** [philc/vimium#3254]
I prefer to disable Vimium for Google Products because they have decent keyboard navigation by themselves; but I fail to do so when I edit Google Slides.

When I try to disable it, it gives current URL as "about:blank" and it still works after I try to block it.

![image](https://user-images.githubusercontent.com/3520704/53297947-7be48f80-3826-11e9-85a5-976fff97ba7c.png)

It seems that it switches to some iframe, because there're a few iframes with such a URL.

Is it a known issue? How can I overcome this?

**Comment by gdh1995** [comment]
Currently, you can add another rule with a pattern of "https://docs.google.com" to also disable Vimium on the main (topest) frame.

### As far as the "about:blank"

Um, this is a *new* issue that I didn't expect. I think Vimium needs to re-think about its exclusion logic: it should also be disabled on such iframes, obeying what it is on the parent/top frame.

**Comment by victortrac** [comment]
This has been driving me crazy for a few days.

**Comment by gdh1995** [comment]
I've released a new version of modified Vimium (I call it Vimium C), and it applies rules from a top frame to such about:blank frames. You may have a try.

Link to download: https://chrome.google.com/webstore/detail/vimium-c/hfjbmagddngcpeloejdejnfgbamkjaeg or https://github.com/gdh1995/vimium-c/releases .

**Comment by srsudar** [comment]
@gdh1995 , do you have any pointers to how you got rules from a top frame applying to `about:blank` frames? I looked through your changelog and tried searching the repo for `about:blank` but didn't see anything immediately obvious.

**Comment by gdh1995** [comment]
Vimium C checks URLs which don't look like `*://*` and turns to apply passkeys of their top frames in:
https://github.com/gdh1995/vimium-c/blob/7e4672cd80ab59ba130e127edde116a6c136640e/background/exclusions.ts#L62-L67

**Comment by binlaniua** [comment]
any update?
